src/@types/apiResponse.d.ts

```diff
@@ -0,0 +1,228 @@
+interface AuthSuccessResponse {
+  key: string;
+}
+
+interface AuthErrorResponse {
+  username?: string[];
+  email?: string[];
+  password?: string[];
+  non_field_errors: string[];
+}
+
+interface GameBackgroundResponse {
```

Comment:
same here

Reply:
For response file i added `Response` in every interface because it will be easier to use this interface at the time of api calls and data formatters
and also if i remove `Response` the name may be similar to some model interface

Reply:
Okay then leave it